acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/utils/BeaconBlockUtil.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2022 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.dsl.utils;
+
+import tech.pegasys.teku.infrastructure.unsigned.UInt64;
+import tech.pegasys.teku.spec.Spec;
+import tech.pegasys.teku.spec.config.SpecConfigBellatrix;
+import tech.pegasys.teku.spec.datastructures.blocks.BeaconBlock;
+import tech.pegasys.teku.spec.datastructures.blocks.blockbody.BeaconBlockBody;
+import tech.pegasys.teku.spec.datastructures.blocks.blockbody.BeaconBlockBodySchema;
+import tech.pegasys.teku.spec.datastructures.execution.ExecutionPayload;
+import tech.pegasys.teku.spec.schemas.SchemaDefinitionsBellatrix;
+import tech.pegasys.teku.spec.util.DataStructureUtil;
+
+import java.util.Random;
+
+import org.apache.tuweni.bytes.Bytes32;
+
+public class BeaconBlockUtil {
+  private final DataStructureUtil util;
+  private int seed = 92892824;
+
+  public BeaconBlockUtil(final Spec spec) {
+    util = new DataStructureUtil(spec);
+  }
+
+  public BeaconBlock randomBeaconBlock(UInt64 slotNum) {
+    UInt64 proposerIndex = util.randomUInt64();
+    Bytes32 previousRoot = util.randomBytes32();
+    Bytes32 stateRoot = util.randomBytes32();
+    BeaconBlockBody body = this.randomBeaconBlockBody(slotNum);
+    return new BeaconBlock(
+        util.getSpec().atSlot(slotNum).getSchemaDefinitions().getBeaconBlockSchema(),
+        slotNum,
+        proposerIndex,
+        previousRoot,
+        stateRoot,
+        body);
+  }
+
+  public BeaconBlockBody randomBeaconBlockBody(final UInt64 slotNum) {
```

Comment:
private?

Reply:
yes, good catch. Resulted due to copy code from Teku.